### PR TITLE
Fixes #12103 - Add rebuild config option for host

### PR DESCRIPTION
**Pull request by shlomizadok**

### CODE REVIEW

**Comment by tstrachota** [comment]
I wanted to test the PR but I'm getting `ERF42-5531 [Foreman::Exception]: unknown permission for api/v2/hosts#rebuild_config` Are there any additional steps I need to do prior to testing this, @shlomizadok ?

File: lib/hammer_cli_foreman/host.rb [hunk 1]
```diff
@@ -514,6 +514,15 @@ def validate_options
       end
     end
 
+    class RebuildConfigCommand < HammerCLIForeman::SingleResourceCommand
+      action :rebuild_config
+      command_name "rebuild-config"
+      desc _('rebuild orchestration related configurations for host')
```
Comment (tstrachota) [inline lib/hammer_cli_foreman/host.rb]: Nitpick: We usually start descriptions with capital letter.

**Comment by tstrachota** [comment]
Apart from the capital letter the code looks good. I still need to test the command though.

**Comment by tstrachota** [comment]
@shlomizadok, I'm still getting the unknown permission exception. Any idea what could be wrong? Migrations and seeds seem to run successfully.

**Comment by shlomizadok** [comment]
@tstrachota - Weird. It works for me...

**Comment by mbacovsky** [comment]
For me it worked with fresh Foreman 1.11.
```
$ hammer -s "https://192.168.121.190/"  host rebuild-config --id 1
Configuration successfully rebuilt.
```

**Comment by mbacovsky** [comment]
:+1:  the code looks okay and works well, thanks @shlomizadok! Merging (with @tstrachota's agreement) :see_no_evil: .

**Pull request merged**